lib/io/storage/StoragePath.js

```diff
@@ -282,13 +293,27 @@ class StoragePath {
     return { namespace: StoragePath.NAMESPACE_REPORTS_STUDY, key };
   }
 
+  static async forStudyRequestReport(report) {
+    const { type, format, ...options } = report;
+
+    const partReportType = StoragePath.getReportType(type);
+    const partTimestamp = StoragePath.getTimestamp();
+    const partOptionsHash = StoragePath.getOptionsHash(options);
+    const { extension } = format;
+    const key = `${partReportType}_${partTimestamp}_${partOptionsHash}.${extension}`;
```

Comment:
`getOptionsHash` already prepends an underscore - we can drop the last underscore here.